build_debian.sh

```diff
@@ -79,7 +79,7 @@ TARGET_PATH=$TARGET_PATH scripts/build_debian_base_system.sh $CONFIGURED_ARCH $I
 # Prepare buildinfo
 sudo scripts/prepare_debian_image_buildinfo.sh $CONFIGURED_ARCH $IMAGE_DISTRO $FILESYSTEM_ROOT $http_proxy
 
-sudo chown root:root $FILESYSTEM_ROOT
+sudo chown -R root:root $FILESYSTEM_ROOT
```

Comment:
do we need to override the whole filesystem recursively?

Reply:
@xumia, could you limited the scope to only `/var/`, or some explicit subfolders under `/var/` ?

Reply:
/var is enough, not necessary to change the whole root /. Thanks.